curriculum/challenges/english/20-upcoming-python/learn-concurrency-by-building-a-prime-number-generator/64e750d9e426ffe0a7c40dfa.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 64e750d9e426ffe0a7c40dfa
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+Asynchronous programming is a programming paradigm that allows tasks to run concurrently, without blocking the execution of other tasks.
+
+In this challenge you would learn how concurrency works by building a prime number generator.
+
+To start, import the `asyncio` library, which provides tools for working with asynchronous code.
```

Comment:
```suggestion
To start, import the built-in `asyncio` module, which provides tools for working with asynchronous code.
```
Python usually uses the _module_ instead of library.